app/components/post_list/post_list_refresh_control.tsx

```diff
@@ -45,7 +48,7 @@ const PostListRefreshControl = ({children, enabled, onRefresh, refreshing, theme
 
     return React.cloneElement(
         children,
-        {refreshControl, inverted: true},
+        {refreshControl, inverted: isInverted},
```

Comment:
any reason why you are adding this? in case of a non-inverted list we should be able to use the normal RefreshControl Approach

Reply:
https://github.com/mattermost/mattermost-mobile/pull/5690#discussion_r714857406 This applies only for the inverted lists?